src/main/java/app/gpx_animator/ui/GeneralSettingsPanel.java

```diff
@@ -824,13 +947,14 @@ protected Type configure(final JFileChooser outputFileChooser) {
         gbcSpeedUnit.gridx = 1;
         add(speedUnitComboBox, gbcSpeedUnit);
 
+
     }
 
     private void setVideoSize(final int width, final int height) {
         widthSpinner.setValue(width);
         heightSpinner.setValue(height);
     }
-
+    
```

Comment:
Just a simple formatting issue: Please correct this whitespace, because you did not modify the source code here.